app/src/main/java/com/capstone/weatherapp/WeatherRepository.kt

```diff
@@ -29,20 +29,15 @@ class WeatherRepository(private val db: WeatherDatabase) {
             _cityListData.postValue(State.Loading)
             try {
                 val cityListResponse = WeatherApiClient.retrofitService.getCities(CITIES, APIKEY, UNITS)
-                if (cityListResponse.isSuccessful) {
-                    val response = cityListResponse.body()!!
-                    _cityListData.postValue(State.Success(response.list))
-                    db.cityDao().insert(response.convertToCityCacheList())
-                }
-                else {
-                    try {
-                        _cityListData.postValue(State.Success(getNetworkCityList()))
-                    } catch (e: Exception) {
-                        _cityListData.postValue(State.Error(e.toString()))
-                    }
-                }
+                val response = cityListResponse.body()!!
+                _cityListData.postValue(State.Success(response.list))
```

Comment:
we don't want to update the value of cityListData yet. We never want to set it to the api response value, we only ever want to set it to the data we fetch from the db so this line can be removed